src/OpenLoco/Objects/ObjectManager.h

```diff
@@ -90,11 +91,51 @@ namespace OpenLoco
     struct competitor_object;
     struct scenario_text_object;
 
-    struct object_repository_item
+#pragma pack(push, 1)
+    struct ObjectHeader
```

Comment:
Move to OpenLoco::S5?

Reply:
Why? It isn't specific to that, it is at the top of every object file.